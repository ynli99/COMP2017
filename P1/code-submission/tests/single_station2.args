"Coffs Harbour" Koolewong "19:45:15"
