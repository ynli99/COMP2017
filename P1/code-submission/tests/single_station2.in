Coffs Harbour::Morisset::19:09:14
Coffs Harbour::Morisset::19:54:14
Coffs Harbour::Morisset::20:01:14
Coffs Harbour::Morisset::20:46:14
Coffs Harbour::Koolewong::19:43:40
Coffs Harbour::Koolewong::20:01:40
