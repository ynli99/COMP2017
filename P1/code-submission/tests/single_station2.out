The next train to Koolewong from Coffs Harbour departs at 20:01:40
